tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "declaration": true,
     "allowSyntheticDefaultImports": true,
     "experimentalDecorators": true,
-    "emitDecoratorMetadata": true
+    "emitDecoratorMetadata": true,
+    "downlevelIteration": true
```

Comment:
Is this still necessary now that we're manually unrolling the iterator into an array?

Reply:
no, this was left in by mistake